frontend/src/views/edit-project/__tests__/reducer.test.ts

```diff
@@ -295,6 +309,38 @@ describe('EditProjectView - reducer', () => {
     });
   });
 
+  it('close one of the two representations', () => {
+    const prevState = projectLoadedAnd2representationsDisplayedState;
+    const action = { type: HANDLE_REPRESENTATION_CLOSED__ACTION, representationId: SECOND_DIAGRAM.id };
+    const state = reducer(prevState, action);
+
+    expect(state).toStrictEqual({
+      viewState: PROJECT_LOADED_AND_REPRESENTATION_DISPLAYED__STATE,
+      project: projectLoadedAndrepresentationDisplayedState.project,
+      representations: projectLoadedAndrepresentationDisplayedState.representations,
+      displayedRepresentation: projectLoadedAndrepresentationDisplayedState.displayedRepresentation,
+      selection: FIRST_DIAGRAM,
+      subscribers: [],
+      message: undefined,
+    });
+  });
+
+  it('close the only representations', () => {
+    const prevState = projectLoadedAndrepresentationDisplayedState;
+    const action = { type: HANDLE_REPRESENTATION_CLOSED__ACTION, representationId: FIRST_DIAGRAM.id };
+    const state = reducer(prevState, action);
+
+    expect(state).toStrictEqual({
+      viewState: PROJECT_LOADED__STATE,
+      project: projectLoadedAndrepresentationDisplayedState.project,
+      representations: [],
+      displayedRepresentation: undefined,
+      selection: undefined,
+      subscribers: [],
+      message: undefined,
+    });
+  });
+
```

Comment:
Could you add a test for the last possibility of your `if / else` in `handleRepresentationClosedAction`?

Reply:
Done